cricketscircle/jekyll-test
# Export story

## How to test

Add content, fill out metadata, click “export story” and you should see a post formatted for Jekyll with the correct filename, front matter and body content.
